PackageInfo.g

```diff
@@ -17,7 +17,7 @@
 SetPackageInfo( rec(
 
 PackageName := "recog",
-Subtitle := "A collection of group recognition methods",
+Subtitle := "A package for constructive recognition of permutation and matrix groups",
```

Comment:
I’m ignorant: is it all constructive, or is there non-constructive stuff as well?

Reply:
Sorry, I’ve finally read the PR description and the linked issue. Happy to accept this subtitle, was just curious.